### ISSUE TITLE
Simulating stoichiometry with unknown coefficients

### ISSUE THREAD

**Issue by janursa** [sys-bio/antimony#59]
this is the request to implement named stoichiometries to allow such an example,
```
model = """
        x = 1;
        y = 10;
        k = .1;
        x_y = 0;
        n=10;
        x + n  y -> x_y; k*x*y^n
    end
"""
```
thanks

**Comment by luciansmith** [comment]
This is now implemented!  You'll be able to use it in release 2.13.0, which I'm working on as we speak.

**Comment by janursa** [comment]
perfect. thank you
